CSS/main.css

```diff
@@ -10,7 +10,7 @@
     height: 800px;
     margin: 0 auto;
 }
-header{
+.main__header{
```

Comment:
Although you're using BEM for the main div and it's direct childrens you're taking it too far. A better approach would be something like this:
`<div class="main">
     <header class="header">
             <h1 class="header__text"> ... </h1>
    </header>
 </div>`